index.bs

```diff
@@ -2367,6 +2374,30 @@ To <dfn>render viewport to a canvas</dfn> given |viewport| and |rect|:
 
 </div>
 
+<div algorithm>
+To <dfn>encode a canvas as Base64</dfn> given |canvas| and |format|:
+
+1. If the |canvas|’s bitmap’s [=origin-clean=] flag is set to false, return
```

Comment:
This seems wrong? apart from anything else I don't think it should be possible since we're creating the canvas as an internal construct, and don't set the flag. But even if it was, it's unclear why it would apply to webdriver.

Reply:
This statement also exists in WebDriver, probably just copy-pasted in. Agreed that this makes no sense.

Reply:
In fact, the bottom statement also doesn't make sense since the canvas is created with positive height and width.